backend/logic/courses/courses.js

```diff
@@ -29,6 +29,43 @@ var queryCourseByTag = async function queryCourseByTag(tag) {
   }
 };
 
+/**
+ * Adds a course into the database
+ * @author Mathieu Savoie
+ * @param {String} courseCode
+ * @param {String} title
+ * @param {String} departement
+ * @returns true if successful
+ * @throws error if MySQL connection failed
+ *         invalid format courses if JSON format is incorrect
+ *         false if insertion failed
+ */
+var addCourse = async (courseCode, title, departement, phasedOut) => {
+  
+  
+  // Verifying proper format
+  await format.verifyCourseCode(courseCode);
+  await format.verifyTitle(title);
+  await format.verifyDepartmentSubName(departement);
+  await format.verifyPhaseOut(phasedOut);
```

Comment:
Same thing here, should be able to create a course in the db without having to hand in a phased out param (also to be corrected in the format.verifyPhasedOut method